helm_chart/ibm-storage-enabler-for-containers/cv-tests/test-default/clean-up.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+###############################################################################
+# Licensed Materials - Property of IBM
+# 5737-E67
+# (C) Copyright IBM Corporation 2016, 2018 All Rights Reserved
```

Comment:
i think it should be 2019 instead of 2016-2018.   ask Tal.

Reply:
ok